{{cookiecutter.extension_name}}/install.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packageManager": "python",
+  "packageName": "{{ cookiecutter.extension_name }}",
+  "uninstallInstructions": "Use your Python package manager (pip, conda, etc.) to uninstall the package {{ cookiecutter.extension_name }}"
```

Comment:
This assumes your javascript name is the same as your python name. Should we assume that? js and python names can be really different.

Reply:
Indeed, thanks for your review. Should we add another entry in the cookiecutter config for the Python extension name?

Reply:
That's what we did in the normal extension cookiecutter.

To update the mimerender cookiecutter, I essentially deleted all the files, copied over the extension cookiecutter, then sort of merged in the two sets of fields and copied the index.ts over. That got me the python package name, etc.

Reply:
There are also other details from `extension-cookiecutter-ts` that would be nice to have here too, such as `outputDir` which is now named `labextension` instead of `static`.

Reply:
Ok thanks I'll do that

Reply:
I did like you said @jasongrout , I started from scratch copying the extension-cookiecutter, thanks for the advice!